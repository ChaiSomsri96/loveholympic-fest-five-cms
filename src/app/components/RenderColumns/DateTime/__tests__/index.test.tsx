import * as React from 'react';
import { render } from '@testing-library/react';

import { DateTime } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<DateTime  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<DateTime />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
