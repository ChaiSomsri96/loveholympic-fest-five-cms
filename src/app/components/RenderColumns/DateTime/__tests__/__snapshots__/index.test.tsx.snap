// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DateTime  /> should match snapshot 1`] = `
<span>
  13/04/2021 22:39 pm
</span>
`;
